src/RestaurantButton.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+export default function RestaurantButton({ orderOneProps }) {
+  return (
+    <div>
+      <button className="btn btn-primary" onClick={orderOneProps}>
```

Comment:
this is good, you're passing your `orderOne` function into your child component, so that when it's called it can update the state in the parent component (this is something we see a lot of in react)